cloud-sql/postgres/knex/test/createTable.test.js

```diff
@@ -22,18 +22,28 @@ const {exec} = require('child_process');
 const cwd = path.join(__dirname, '..');
 
 const {DB_USER, DB_PASS, DB_NAME} = process.env;
-const CONNECTION_NAME = process.env.CLOUD_SQL_CONNECTION_NAME;
-
+const CONNECTION_NAME = process.env.INSTANCE_CONNECTION_NAME;
+const DB_HOST = process.env.DB_HOST;
+const SOCKET_PATH = process.env.DB_SOCKET_PATH || "/cloudsql"
 before(async () => {
+  const connection = {
+    user: DB_USER,
+    password: DB_PASS,
+    database: DB_NAME,
+  };
+
+  if (DB_HOST) {
+    const dbSocketAddr = process.env.DB_HOST.split(":");
+    connection.host = dbSocketAddr[0];
+    connection.port = dbSocketAddr[1];
+  } else {
+    connection.host = `${SOCKET_PATH}/${CONNECTION_NAME}`;
+  }
+
   try {
     const knex = Knex({
       client: 'pg',
-      connection: {
-        user: DB_USER,
-        password: DB_PASS,
-        database: DB_NAME,
-        host: `/cloudsql/${CONNECTION_NAME}`,
-      },
+      connection: connection,
```

Comment:
I don't recall the practice in this repo, should this use object property shorthand?

Reply:
I updated it in the latest commit, but I'm not sure what the repo standard is.